sql/localsql/nipost/nipost.go

```diff
@@ -16,7 +17,7 @@ type NIPostState struct {
 }
 
 func AddNIPost(db sql.Executor, nodeID types.NodeID, nipost *NIPostState) error {
-	buf, err := codec.Encode(&nipost.Membership)
+	buf, err := codec.Encode(nipost.Membership.ToWireV1())
```

Comment:
I think originally I did this because I was lazy and didn't want to map the `Membership` object to columns of the table, which I still think is more effort than what it is worth.

Yet we must not have any wire types in our `sql` functions, so I believe we need a dedicated type here, that is only used to serialize the membership for storing the DB and deserialze it when fetching it again. We cannot have our `sql` package depend on actual wire types.